DynamicLights/SpikingNet.cpp

```diff
@@ -713,6 +717,19 @@ bool SpikingNet::getFlagDecay() const {
     return this->flagDecay;
 }
 
+void SpikingNet::writeNetworkType(SpikingNet::NetworkType networkType) {
+    if (this->networkType == networkType)
+        return;
+
+    // reset network, since these parameters only take effect when the network is created anew
+    // note from Julien: shouldn't the network be reinitialized AFTER changing the values?
+    reset();
+    initialize();
+
+    this->networkType = networkType;
+    emit networkTypeChanged(networkType);
+}
+
```

Comment:
Network types will get deprecated when we implement the drawing interface, since the only network type that will work with this is the grid network. We don't need to have them be an editable property or be part of the UI at the moment.

Reply:
Also, your comment in this snippet made me realize the reset / init sequence is wrong. It should be reset, change, init, emit, as you suggested.

Reply:
noted, thanks! I felt like this property was the easiest gateway for me to test exposing enums from C++ to QML in order to populate dropdown fields, I didn't fully consider it would be unnecessary to include this property in the GUI altogether. will fix ASAP